create3_lidar/README.md

```diff
@@ -57,15 +65,22 @@ ros2 run teleop_twist_keyboard teleop_twist_keyboard
 
 ### Computer Setup
 
-On your computer, start rviz2.
-(TODO: more to come here)
+![Image of rviz with custom configuration](https://iroboteducation.github.io/create3_docs/examples/data/create3_lidar_rviz.png)
+
+On your computer, start rviz2. If you run it from a cloned version of this repository, you will find an rviz configuration installed.
```

Comment:
Maybe, something along the lines of:
"If you install this package on your host machine where you run `rviz2`, it will put create3_lidar.rviz into install/share/create3_lidar/rviz.  From rviz2, you can use the menu bar to select "File"->"Open Config" then select the create3_lidar.rviz file.  This file will configure rviz2 to subscribe to the laser, the occupancy map, and display the `base_footprint` tf frame that the laser is building off of from the map frame."